### ISSUE TITLE
Writing definitions creates infinite restarting loop

### ISSUE THREAD

**Issue by moltar** [nestjs/graphql#87]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Providing a `definitions` option that writes a TypeScript class:

```
      definitions: {
        path: path.resolve(__dirname, 'app.graphql.ts'),
        outputAs: 'class',
      },
```

This works as expected, but when running in dev mode, will keep restarting, because on every start the file is written and the change is detected by Nodemon.

## Expected behavior

No restart is necessary.

## Minimal reproduction of the problem with instructions

Add the following config option:

```
      definitions: {
        path: path.resolve(__dirname, 'app.graphql.ts'),
        outputAs: 'class',
      },
```

Then start in dev mode:

```
npm run start:dev
```

## What is the motivation / use case for changing the behavior?

To prevent endless restarts.

## Environment

<pre><code>
[System Information]
OS Version     : macOS
NodeJS Version : v10.5.0
NPM Version    : 6.4.1
[Nest Information]
graphql version : 5.4.0
common version  : 5.4.0
core version    : 5.4.0</code></pre>

**Comment by moltar** [comment]
This, of course, can be fixed by adding the file to Nodemon ignore:

```
  "ignore": ["src/**/*.spec.ts", "src/app.graphql.ts"],
```

Maybe needs to be documented?

**Comment by kamilmysliwiec** [comment]
Should be fixed due to #81

**Comment by val-samonte** [comment]
For the people who are using angular + nrwl nx and running ng serve, be sure to omit any comments on the schema file, as the lib will compare if there are changes and the comments were not removed.

eg. from the example (graphql.schema.ts)
`/* tslint:disable */ <--- remove this`
